db/migrate/20160629122413_add_database_name_to_endpoints.rb

```diff
@@ -0,0 +1,5 @@
+class AddDatabaseNameToEndpoints < ActiveRecord::Migration[5.0]
+  def change
+    add_column :endpoints, :database_name, :string
```

Comment:
👍 @fryguy

Sent from my iPhone

> On 12 Jul 2016, at 22:13, Jason Frey <notifications@github.com> wrote:
> 
> In db/migrate/20160629122413_add_database_name_to_endpoints.rb:
> 
> > @@ -0,0 +1,5 @@
> > +class AddDatabaseNameToEndpoints < ActiveRecord::Migration[5.0]
> > +  def change
> > +    add_column :endpoints, :database_name, :string
> I don't really like this idea. I want the URL field to ✨ go away ✨. I guess my question before giving my full opinion is if the database_name is actually a part of the URL or not. If it is part of the URL, then it should be the appropriate column for the URL part (i.e. if it's the path part of the URL, then it should go in path). Otherwise, I think I'd prefer a generic options/settings hash column to allow for "other options".
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>